happi/tests/test_cli.py

```diff
@@ -79,6 +83,16 @@ def test_cli_version(capsys):
     assert happi.__file__ in readout.out
 
 
+def test_cli_no_argument(capsys):
+    happi.cli.happi_cli([])
+    readout = capsys.readouterr()
+    # TODO: figure out a way to get usage: happi
+    # instead of usage: pytest or usage run_test.py..
+    assert 'usage:' in readout.out
+    assert '[-h] [--path PATH] [--verbose] [--version]\n'
+    '                    {search,add,edit,load} ...\n' in readout.out
```

Comment:
I think this isn't part of the assert...
```suggestion
    assert '{search,add,edit,load}' in readout.out
```

Reply:
Changed this in 8d8698a - thank you